apps/drec-api/src/pods/device/device.service.ts

```diff
@@ -73,54 +75,33 @@ export class DeviceService {
     return await this.repository.save(currentDevice);
   }
 
-  private getFilteredQuery(filterDto: FilterDTO): FindManyOptions<Device> {
-    const {
-      fuel_code,
-      device_type_code,
-      installation_configuration,
-      capacity,
-      startDate,
-      endDate,
-      grid_interconnection,
-      off_taker,
-      sector,
-      labels,
-      standard_compliance,
-    } = filterDto;
-    let where: FindConditions<Device> = {};
-    if (fuel_code) {
-      where.fuel_code = fuel_code;
-    }
-    if (device_type_code) {
-      where.device_type_code = device_type_code;
-    }
-    if (installation_configuration) {
-      where.installation_configuration = installation_configuration;
-    }
-    if (capacity) {
-      where.capacity = capacity;
-    }
-    if (grid_interconnection) {
-      where.grid_interconnection = grid_interconnection;
-    }
-    if (off_taker) {
-      where.off_taker = off_taker;
-    }
-    if (sector) {
-      where.sector = sector;
-    }
-    if (labels) {
-      where.labels = ILike(`%${labels}%`);
-    }
-    if (standard_compliance) {
-      where.standard_compliance = standard_compliance;
-    }
-    if (startDate && endDate) {
-      where.commissioning_date = Between(startDate, endDate);
-    }
+  private getFilteredQuery(filter: FilterDTO): FindManyOptions<Device> {
+    const where: FindConditions<Device> = cleanDeep.default({
```

Comment:
em, so if it's default export, why not `import cleanDeep from 'clean-deep"` ?

Reply:
good point